events/_event-post-macros.html

```diff
@@ -0,0 +1,311 @@
+
+
+{# ==========================================================================
+
+   calendar_icon()
+
+   ==========================================================================
+
+   Description:
+
+   Create an HTML calendar icon when given:
+
+   calendar_date:      A valid date.
+
+   additional_classes: Extra classes you wish to add to the icon, space
+                       separated.
+
+   ========================================================================== #}
+
+{% macro calendar_icon(calendar_date, additional_classes) %}
+    <time class="calendar-icon {{ additional_classes }}" data-month="{{ calendar_date | date('%b') }}" data-day="{{ calendar_date | date('%d') }}" datetime="{{ calendar_date }}">
+        <span class="u-visually-hidden">{{ calendar_date | date('%b %d, %Y') }}</span>
+    </time>
+{% endmacro %}
+
+
+
+{# ==========================================================================
+
+   event_location_image()
+
+   ==========================================================================
+
+   Description:
+
+   Create HTML image when given:
+
+   post: A post from a query result.
+
+   options (optional): An object to use for customizing a event location image using
+                       Google Maps.
+
+   options.class:    A class name to be applied to the img element.
```

Comment:
Yeah, should be:
additional_classes: Extra classes you wish to add to the icon, space separated.